### PR TITLE
Add default config to config page UI

### PR DESCRIPTION
**Pull request by nandhinibsn**
Resolve #

## Description:

This PR is to call the default config API from the back end to display the default config subfields in scenarios like postprocessor or similarity being null
## Checklist:

You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

File: webapp/src/pages/Settings.tsx [hunk 1]
```diff
@@ -179,15 +174,16 @@ const NumberField: React.FC<
...
   const [updateConfig] = updateConfigEndpoint.useMutation();
 
   const [partialConfig, setPartialConfig] = React.useState<
     Partial<AzimuthConfig>
   >({});
 
-  // If config was undefined, PipelineCheck would not even render the page.
-  if (config === undefined) return null;
+  // If config or defaultConfig was undefined, PipelineCheck would not even render the page.
+  if (config === undefined || defaultConfig === undefined) return null;
```
Comment (JosephMarinier) [inline webapp/src/pages/Settings.tsx:186]: That's not true for `defaultConfig`. We should revert that change and handle loading/errors with `defaultConfig`. We'll need `isLoading` and `error` from
```diff
-  const { data: defaultConfig } = getDefaultConfigEndpoint.useQuery({ jobId });
+  const { data: defaultConfig, isLoading, error } = getDefaultConfigEndpoint.useQuery({ jobId });
```
and
```suggestion
  // If config was undefined, PipelineCheck would not even render the page.
  if (config === undefined) return null;

  if (isLoading) {
    return <Loading />;
  } else if (error || defaultConfig === undefined) {
    return (
      <Box alignItems="center" display="grid" justifyItems="center">
        <img src={noData} width="50%" alt="No confusion matrix data" />
        <Typography>{error?.message || UNKNOWN_ERROR}</Typography>
      </Box>
    );
  }
```

Comment (nandhinibsn) [reply webapp/src/pages/Settings.tsx:186]: Okay, Yes, I agree with the importance of it. Thanks

Comment (JosephMarinier) [reply webapp/src/pages/Settings.tsx]: Don't forget to revert the comment as I suggested.

Comment (nandhinibsn) [reply webapp/src/pages/Settings.tsx]: Okay, Sorry I missed out that part. I will add it.Thanks

**Review by JosephMarinier — APPROVED** [review]
Good job! Works well! 👍

**Comment by nandhinibsn** [comment]
Thank you!!

**Pull request merged**